CHANGELOG.md

```diff
@@ -152,6 +152,32 @@
 * Added `InterpolationMethod` and `HueInterpolationMethod` which collectively
   represent the method to use to interpolate two colors.
 
+### JS API
+
+* Modify constructor to accept a new `space` option, with support for all the
```

Comment:
Replace "constructor" with the name of the constructor.

Reply:
Addressed in https://github.com/sass/dart-sass/pull/2117/commits/63c17e31c3f5f6b168bba3f32ecc33ac65113593